src/index.ts

```diff
@@ -55,6 +94,14 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 
 			await emit('apiImpl', path.join(outputPath, relativeSourceOutputPath, 'impl', `${context.generator().toIdentifier(group.name)}.ts`), 
 				{ ...rootContext, ...group, ...doc }, false, hbs)
+
+			for (const operation of operations) {
```

Comment:
It's a little confusing to use a loop here when we're really just looking to see if any of the operations use a multipart request... how about you refactor the algorithm out of `isAnyOperationSupportMultipart` into a function that we can use here and there.